java/src/jmri/jmrit/whereused/Bundle.properties

```diff
@@ -10,7 +10,7 @@ SaveButton                    = Save to Text File
 SaveButtonHint                = Save the where used content to a text file
 
 ItemTypeNone = - Select Type -
-TypePrompt = \tSelect an item type, then select an item name and then press Create.\t
+TypePrompt = tSelect an item type, then select an item name and then press [{0}].t
```

Comment:
Is this change correct? Just checking. I suppose `\t` is TAB character, but `tSelect` and `[{0}].t` seems wrong, so I just ask to check.

Reply:
Correct. The `\t` escapes disappeared as I edited this line in the IDE Bundle view. Handy for comparisons but risky where we use special chars. Will put them back in editing the simple file view.

Reply:
Fixed